R/config.yml

```diff
@@ -1,10 +1,7 @@
-# c("user_id", "update_ts", "client", "curr_platform", "n_trips",
-# "n_trips_today", "n_active_days", "first_trip_datetime", "last_trip_datetime",
-# "first_trip_local_datetime", "last_trip_local_datetime", "n_days",
-# "first_get_call", "last_get_call", "first_put_call", "last_put_call",
-# "first_diary_call", "last_diary_call")
+# Use this file to set options for the app
 
 default:
```

Comment:
This file should be stored in `inst` folder.

Reply:
Fixes in https://github.com/asiripanich/emdash/pull/32/commits/c9aeb173bc098669dc37c82d0f03d412780b6e3c